ci/release/changelogs/next.md

```diff
@@ -28,3 +28,4 @@
 - Fixes crash when using `--watch` and navigating to an invalid board path [#1693](https://github.com/terrastruct/d2/pull/1693)
 - Fixes edge case where nested edge globs were creating excess shapes [#1713](https://github.com/terrastruct/d2/pull/1713)
 - Fixes a panic with a connection to a grid cell that is a container in TALA [#1729](https://github.com/terrastruct/d2/pull/1729)
+- Fixes positioning of grid cells when the grid is a shape with a custom innerBox and fixes the circle shape's content sometimes not fitting in its innerBox. [#1734](https://github.com/terrastruct/d2/pull/1734)
```

Comment:
this leaks implementation details. changelog should make sense to users. rephrase in way that's "here's the value added"